dashboard/src/main/webapp/src/components/shufflecomponent/ActiveNodeListPage.vue

```diff
@@ -20,14 +20,14 @@
     <el-table :data="pageData.tableData" height="550" style="width: 100%">
       <el-table-column prop="id" label="Id" min-width="180"/>
       <el-table-column prop="ip" label="IP" min-width="80"/>
-      <el-table-column prop="grpcPort" label="Port" min-width="80"/>
+      <el-table-column prop="grpcPort" label="GrpcPort" min-width="80"/>
+      <el-table-column prop="nettyPort" label="NettyPort" min-width="80"/>
       <el-table-column prop="usedMemory" label="UsedMem" min-width="80"/>
       <el-table-column prop="preAllocatedMemory" label="PreAllocatedMem" min-width="80"/>
       <el-table-column prop="availableMemory" label="AvailableMem" min-width="80"/>
-      <el-table-column prop="totalMemory" label="TotalMem" min-width="80"/>
       <el-table-column prop="eventNumInFlush" label="FlushNum" min-width="80"/>
       <el-table-column prop="status" label="Status" min-width="80"/>
-      <el-table-column prop="timestamp" label="ResigerTime" min-width="80"/>
+      <el-table-column prop="timeString" label="ResigerTime" min-width="80"/>
```

Comment:
Could we use `formatter`?

Reply:
> Could we use `formatter`?

The back end simply returns the formatted string, no formatter required.

Reply:
I mean we can remove `ServerNodeVO` and use `formatter` to implement this function.

Reply:
> I mean we can remove `ServerNodeVO` and use `formatter` to implement this function.

This class also has the function of unit conversion display format, VO object is specially used to do front-end display formatting, does not affect other functions.

Reply:
But there will be more redundant code. @jerqi @smallzhongfeng WDYT?

Reply:
I'm not familiar about this. I refer to Spark's implement, too. Spark doesn't use formatter, too.  So I think we keep this.

Reply:
https://element.eleme.io/#/zh-CN/component/table

Reply:
Ok, let me correct it

Reply:
It has been changed to the same native display form as Spark, if you need to optimize the display later on the front end.

Reply:
> if you need to optimize the display later on the front end.

Could you optimize this in this pr?

Reply:
> > if you need to optimize the display later on the front end.
> 
> Could you optimize this in this pr?

Have been added.